src/fundamental/tpm-pcr.h

```diff
@@ -31,6 +31,7 @@ typedef enum UnifiedSection {
         UNIFIED_SECTION_DTB,
         UNIFIED_SECTION_PCRSIG,
         UNIFIED_SECTION_PCRPKEY,
+        UNIFIED_SECTION_UNAME,
         _UNIFIED_SECTION_MAX,
 } UnifiedSection;
 
```

Comment:
please also update the table in make_uki() in ukify.py that lists the sections covered by measurement